docs/cudf/source/basics.rst

```diff
@@ -0,0 +1,54 @@
++------------------------+------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+| Float                  |                  | np.float32_, np.float64_                                                            | ``'float32'``, ``'float64'``                |
++------------------------+------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+| Strings                |                  | `str <https://docs.python.org/3/library/stdtypes.html#str>`_                        | ``'string'``, ``'object'``                  |
++------------------------+------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+| Datetime               |                  | np.datetime64_                                                                      | ``'datetime64[s]'``, ``'datetime64[ms]'``,  |
+|                        |                  |                                                                                     | ``'datetime64[us]'``, ``'datetime64[ns]'``  |
++------------------------+------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+| Timedelta              |                  | np.timedelta64_                                                                     | ``'timedelta64[s]'``, ``'timedelta64[ms]'``,|
+| (duration type)        |                  |                                                                                     | ``'timedelta64[us]'``, ``'timedelta64[ns]'``|
++------------------------+------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+| Categorical            | CategoricalDtype | (none)                                                                              | ``'category'``                              |
++------------------------+------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+| Boolean                |                  | np.bool_                                                                            | ``'bool'``                                  |
++------------------------+------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+| List                   |  ListDtype       | `list <https://docs.python.org/3/tutorial/datastructures.html#more-on-lists>`_      | ``'list'``                                  |
```

Comment:
Should the string alias here be `List(type)` since we expected a capital `L` as well as expected the child type supplied as well?

Reply:
Yes, after looking at the code I should have added `ListDtype("int"), .., ..etc` but  Looks like `ListDtype` is missing docs and not exposed in sphinx-docs, should I add expose it for `0.15`? Is there some limitation of note/disclaimer to be made for the support of the `ListDtype`?

Reply:
Lets not expose it for `0.15` as the support is extremely early / not really supported well yet.

Reply:
In that case, should I remove the entry from dtypes table?

Reply:
I would propose we do that for 0.15 and we can add it in 0.16.